content/intro-to-storybook/svelte/en/using-addons.md

```diff
@@ -0,0 +1,235 @@
+---
+title: 'Addons'
+tocTitle: 'Addons'
+description: 'Learn how to integrate and use addons using a popular example'
+---
+
+Storybook boasts a robust system of [addons](https://storybook.js.org/addons/introduction/) with which you can enhance the developer experience for
+everybody in your team. If you've been following along with this tutorial linearly, we have referenced multiple addons so far, and you will have already implemented one in the [Testing chapter](/svelte/en/test/).
+
+<div class="aside">
+<strong>Looking for a list of potential addons?</strong>
+<br/>
+😍 You can see the list of officially-supported and strongly-supported community addons <a href="https://storybook.js.org/addons/addon-gallery/">here</a>.
+</div>
+
+We could write forever about configuring and using addons for all of your particular use-cases. For now, let's work towards integrating one of the most popular addons within Storybook's ecosystem: [knobs](https://github.com/storybooks/storybook/tree/master/addons/knobs).
+
+## Setting Up Knobs
+
+Knobs is an amazing resource for designers and developers to experiment and play with components in a controlled environment without the need to code! You essentially provide dynamically defined fields with which a user manipulates the props being passed to the components in your stories. Here's what we're going to implement...
+
+<video autoPlay muted playsInline loop>
+  <source
+    src="/intro-to-storybook/addon-knobs-demo.mp4"
+    type="video/mp4"
+  />
+</video>
+
+### Installation
+
+First, we will need to install all the necessary dependencies.
+
+```bash
+npm install @storybook/addon-knobs
```

Comment:
This should be `-D`

Reply:
It's now updated with the latest commit. All of the other tutorials should be also updated to reflect this changes, as they are instructing to add the addon as a dependency. Going to mark it as resolved if you don't mind